scholarx/archive/functions.js

```diff
@@ -2,6 +2,10 @@ $(function () {
     loadNavAndFooter('/assets/content/static');  //relative path to content directory
 });
 
+//variables to hold mentor and mentee data
+let mentor2019, mentor2020, mentor2021;
```

Comment:
Instead of keeping variables, we can maintain an object. Something like this.
```
 let mentors = {
    2019: [],
    2020: []
 }
```

Reply:
@anjula-sack @Madhawa97 Can you check now guys?